src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ShowPlan/OperationTable.cs

```diff
@@ -259,7 +259,7 @@ static OperationTable()
                 new Operation("DeletedScan",            SR.Keys.LogicalOpDeletedScan),
                 new Operation("DistinctSort",           SR.Keys.LogicalOpDistinctSort),
                 new Operation("Distinct",               SR.Keys.LogicalOpDistinct),
-                new Operation("DistributeStreams",      SR.Keys.LogicalOpDistributeStreams, SR.Keys.DistributeStreamsDescription, "Parallelism_distribute.ico"),
+                new Operation("DistributeStreams",      SR.Keys.LogicalOpDistributeStreams, SR.Keys.DistributeStreamsDescription, "parallelismDistribute"),
```

Comment:
Why are there only a couple of these with these additional parameters? Especially since the physical operations seem to all have the 4 parameters.

Reply:
I am guessing there isn't an icon for other operations. This code is ported from SSMS so I do not have the original  context about it.

Reply:
I would highly encourage finding that information out and documenting it. Even if you don't know the original "reason" for why you should still find out what the result is and try to reason out why it's done that way (for example the main follow up question I would have is why isn't there an icon for these other operations?)